docs/CONFIG-PROPERTIES.md

```diff
@@ -24,7 +24,7 @@
 | timer.port.testinterval | timer in seconds | 300 | retest the current port config |
 | timer.port.timeout | timer in seconds | 15 | time for each http/send |
 | timer.port.testbetterinterval | timer in seconds | 600 | test a higher prio port config |
-| network.fallback.any.eth | "enabled" or "disabled" | enabled (disabled if device was installed with [bootstrap config](CONFIG.md) included) | if no connectivity try any Ethernet, WiFi, or LTE |
+| network.fallback.any.eth | "enabled" or "disabled" | disabled (enabled forcefully but only temporarily when device has no network config available) | if no connectivity try any Ethernet, WiFi, or LTE with DHCP client |
```

Comment:
"no network config available" almost reads like "no network available".
Maybe "enabled forcefully during onboarding if no network config" is more clear?